docs/backup-modules.md

```diff
@@ -1,77 +1,64 @@
+adding a leading dot to the filename; this is done to avoid that it
+is displayed as a normal module in `module avail`.
+- With module files in Lua syntax, the backup module file is not made
+hidden (unless Lmod 6.x is used), since the additional `.bak_*`
+extension prevents from picking it up as a valid module file; only
+files ending in `.lua` are considered to be module files by Lmod 7+.
 
-```rst
-.. _backup_modules:
-
-Backing up of existing modules (``--backup-modules``)
------------------------------------------------------
-
-While regenerating existing module files, you may want to preserve the existing module files
-to compare and assess that the changes in the newly generated module file match expectations.
-
-Backing up of existing modules can be enabled with ``--backup-modules``.
-
-Backups are stored in the same directory as where the module file was located,
-and the files are given an additional extension of the form ``.bak_<year><month><day><hour><min><sec>``.
-
-* With module files in Tcl syntax, the backup module file is hidden by adding a leading dot to the filename;
-  this is done to avoid that it is displayed as a normal module in ``module avail``.
-* With module files in Lua syntax, the backup module file is not made hidden (unless Lmod 6.x is used),
-  since the additional ``.bak_*`` extension prevents from picking it up as a valid module file;
-  only files ending in ``.lua`` are considered to be module files by Lmod 7+ .
-
-The location of the backed up module file will be printed, as well as a "unified diff" comparison
-(very similar to what ``diff -u`` produces) between the backed up module file and the newly generated module file
+The location of the backed up module file will be printed, as well as a
+"unified diff" comparison (very similar to what `diff -u` produces)
+between the backed up module file and the newly generated module file
 (or a message mentioning that no differences were found).
 
+## Disabling automatic backup of modules
 
-.. _backup_modules_disable:
-
-Disabling automatic backup of modules
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-When ``--skip`` or ``--module-only`` is used, backing up of existing modules is enabled automatically.
+When `--skip` or `--module-only` is used, backing up of existing modules
+is enabled automatically.
 
-This can be disabled with ``--disable-backup-modules``.
+This can be disabled with `--disable-backup-modules`.
 
+## Example
 
-.. _backup_modules_example:
-
-Example
-~~~~~~~
-
-Suppose existing modules in both Tcl & Lua syntax are present (``bzip2/1.0.6``).
+Suppose existing modules in both Tcl & Lua syntax are present
+(`bzip2/1.0.6`).
 
 Both these module files are missing an update statement for $PATH
-because the ``/bin`` subdirectory was missing in the installation,
-for the purpose of this example::
+because the `/bin` subdirectory was missing in the installation, for the
+purpose of this example:
 
+!!! example ""
```

Comment:
I wouldn't use [admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions) for these examples, those are more for notes and warnings imho, just a regular `shell` code block (see https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#usage).

Reply:
@boegel with these blocks would you like `bash` syntax highlighting?